app/frontend/wallet/shelley/shelley-ledger-crypto-provider.ts

```diff
@@ -166,7 +169,7 @@ const ShelleyLedgerCryptoProvider = async ({
 
   async function displayAddressForPath(
     absDerivationPath: BIP32Path,
-    stakingPath?: BIP32Path
+    stakingPath: BIP32Path
```

Comment:
stakingPath being optional seems to be a left-over from byron era and the general CryptoProvider interface had it actually enforced